Update SUSE Copyright years

Update the SUSE copyright range for changes made in 2020 and 2021.

2021-02-26  Tom de Vries  <tdevries@suse.de>

	* dwz.c: Extend SUSE Copyright range to 2021.
	* COPYRIGHT_YEARS: Regenerate.

---
 COPYRIGHT_YEARS | 2 +-
 dwz.c           | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/COPYRIGHT_YEARS b/COPYRIGHT_YEARS
index fdd9942..e4b1511 100644
--- a/COPYRIGHT_YEARS
+++ b/COPYRIGHT_YEARS
@@ -1,3 +1,3 @@
 -DFSF_YEARS='"1992-2021"'
 -DRH_YEARS='"2001-2021"'
--DSUSE_YEARS='"2019"'
+-DSUSE_YEARS='"2019-2021"'
diff --git a/dwz.c b/dwz.c
index 3342aba..bc37f32 100644
--- a/dwz.c
+++ b/dwz.c
@@ -1,6 +1,6 @@
 /* Copyright (C) 2001-2021 Red Hat, Inc.
    Copyright (C) 2003 Free Software Foundation, Inc.
-   Copyright (C) 2019 SUSE LLC.
+   Copyright (C) 2019-2021 SUSE LLC.
    Written by Jakub Jelinek <jakub@redhat.com>, 2012.
 
    This program is free software; you can redistribute it and/or modify
